javascript/modules/client/src/internal/interfaces/plugins.ts

```diff
@@ -133,3 +159,28 @@ export enum ProposalCreationSteps {
 export type ProposalCreationStepValue =
   | { key: ProposalCreationSteps.CREATING; txHash: string }
   | { key: ProposalCreationSteps.DONE; proposalId: string };
+
+// PROPOSAL RETRIEVAL
+
+export interface IErc20VotingProposal extends IProposal {
+  voteId: string;
+  token: DaoToken;
+
+  yea?: number;
+  nay?: number;
+  abstain?: number;
+
+  open: boolean;
+  participationRequiredPct: number;
+  supportRequiredPct: number;
+  votingPower: number;
+
+  voters: { id: string; voterState: VoteOptions; weight: number }[];
+}
```

Comment:
As stated above, I would recommend using `types` for strict data and do `... = Proposal & { ... }` 

Also, adapt the modifications suggested before in the code